frontend/src/views/WheresMyGene/components/CellInfoSideBar/QuestionMarkIconSvg.tsx

```diff
@@ -0,0 +1,7 @@
+const QuestionMarkIconSvg = (): JSX.Element => (
```

Comment:
you should be able to import the svg directly instead of writing raw svg in tsx. checkout the reset button after deleting genes/cell types or the info button next to cell type labels for an example.

Reply:
Check out what I did [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/42d0b23e4610d2b64c2b9c2caaec74e679236586/frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/style.ts#L103).

If the color change is gray-scale, then we can mess with the brightness prop to do it. For consistency with what is done for the info buttons, I would actually just apply the same style and turn the button black on hover.

Reply:
Cool! I just made that change, check it out when you can!